langchain.gemspec

```diff
@@ -56,6 +56,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "qdrant-ruby", "~> 0.9.0"
   spec.add_development_dependency "ruby-openai", "~> 4.0.0"
   spec.add_development_dependency "safe_ruby", "~> 1.0.4"
+  spec.add_development_dependency "sequel", "~> 5.68.0"
+  spec.add_development_dependency "sqlite3", "~> 1.6.3"
```

Comment:
I think we can get rid of the `sqlite3` gem, right?

Reply:
Fixed.